t/16-utils.t

```diff
@@ -23,6 +23,15 @@ BEGIN {
 use strict;
 use OpenQA::Utils;
 use Test::More;
+use FindBin '$Bin';
+
+my $changelog_file  = $Bin . '/../public/Changelog';
+my $refs_file       = $Bin . '/../.git/packed-refs';
+my $head_file       = $Bin . '/../.git/refs/heads/master';
```

Comment:
I guess ideally you'd want a separate sandbox directory for each of these cases, to get the best test coverage.